bdns/dns.go

```diff
@@ -116,7 +116,7 @@ var (
 
 // DNSResolver defines methods used for DNS resolution
 type DNSResolver interface {
-	LookupTXT(string) ([]string, error)
```

Comment:
You're adding a new, same-type entry to this, so yeah it is. Don't need to do it for all of them.